PF/Assignment1/VehicleReadMain.java

```diff
@@ -0,0 +1,80 @@
+import java.util.List;
+import java.util.Scanner;
+
+//Main class to perform all operations
+public class VehicleReadMain {
+	public static void main(String args[]) throws VehicleManagementException {
```

Comment:
ACTION: There is no class present with name VehicleManagementException. Have you complied this code before commit?